### PR TITLE
Note client impact of `fetch` implementation change in CHANGELOG

### LINKED ISSUE #129: Change log should warn about side effects of moving from `XMLHttpRequest` to `fetch in `1.10.0`
In release 1.10.0, we changed the underlying transported used in browsers from `XMLHttpRequest` to `fetch`:

https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/main/CHANGELOG.md#v1100

If prior to 1.10.0, you were passing `fetchOptions: {redirect: 'manual'}` as a server-side performance optimization to prevent following then this upgrade can cause a regression as this option is silently ignored by `XMLHttpRequest`, but DOES impact `fetch`.

Commonly, this would cause SLAS `authorizeCustomer` requests to fail with `opaqueredirect` errors.

Folks upgrading should either:

a) remove these optimizations and use the provided helper methods provided by the SDK which implement them or ...
b) update their code to only provide `fetchOptions.redirect = 'manual'` server side (and not client side)

### PR DESCRIPTION
**Pull request by bfeister**
Closes #129

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #### Bug fixes
 
 - The 'To is not a function' [issue](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/issues/122) is fixed.
+- Migrated from `cross-fetch` to `node-fetch` via [this PR](https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/commit/27609f35def0494db7ec45fb948a18c0eac2d25e) which has potential impact for client implementations. Passing `fetchOptions: {redirect: 'manual'}` as called on the server-side to prevent following of redirects prior to this upgrade had no effect on the client because `cross-fetch` used `XMLHttpRequest` and was silently ignored. The new approach respects this `redirect: 'manual'` directive, potentially causing redirects on the client (given these optional parameters) to not be followed, which breaks flows including the SLAS `authorizeCustomer` method, which could fail with an `opaqueredirect` error.
```
Comment (alexvuong) [inline CHANGELOG.md:8]: Should we put this info into README instead?

Comment (bfeister) [reply CHANGELOG.md:8]: I think because it's a migration detail, CHANGELOG seems appropriate

**Review by alexvuong — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
...
 
 ## v1.10.0
```
[the remark this answers is not in the document]
Comment (bfeister) [reply CHANGELOG.md:10]: Good call, though we need to merge that back to `v1.10.1` right? I think it's ultimately two PRs isn't it?

[the remark this answers is not in the document]
Comment (bfeister) [reply CHANGELOG.md:10]: That sounds right @wjhsf , thank you. I think we can move ahead @joeluong-sfcc

**Comment by bfeister** [comment]
@wjhsf @joeluong-sfcc can I get approval on this?

**Review by joeluong-sfcc — CHANGES_REQUESTED** [review]
Could you move the notes under this point in the `CHANGELOG` pretty please 🥺:

https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/blob/18ddc203024c888199ca9f310d90bffab09930ed/CHANGELOG.md?plain=1#L14 

This description really should be under the `v1.10.0` section

**Comment by joeluong-sfcc** [comment]
Closing as this PR takes care of this: https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/pull/132

Also updated in release notes for `v1.10.0`:  https://github.com/SalesforceCommerceCloud/commerce-sdk-isomorphic/releases/tag/v1.10.0

**Pull request closed without merging**